docs/faq.rst

```diff
@@ -0,0 +1,112 @@
+    different subgroups.
+    Mitigation algorithms seek to improve the fairness metrics without strongly affecting
+    the accuracy, or more generally to navigate the trade-offs between performance and
+    fairness metrics.
+
+Can the mitigation algorithms in Fairlearn make my model fair?
+    Naive use of the Fairlearn library is not going to result in fairer models.
+    Indeed, when used incorrectly, these mitigation algorithms could produce a model which is *less* fair.
+    What mitigation algorithms do is generate a model which has lower disparity as measured by
+    a user-specified metric.
+    Whether this makes the model fairer is outside the scope of the algorithm.
+    To determine whether the new model is fairer, the whole system of which it is a part
+    needs to be considered in terms of the societal context in which it is used.
+    In Fairlearn, we aim to combine a Python library of disparity metrics and mitigation algorithms
+    with a deep discussion of fairness in machine learning.
+    By considering this extra information in the context of your particular machine learning system,
+    you should be able to determine the most appropriate actions for your situation.
+
+What sort of fairness related harms can the Fairlearn library address?
+    Fairlearn concentrates on group fairness, meaning obtaining the minimum disparity on some
+    metric evaluated on specified subgroups in the data.
+    This is certainly not the only way of looking at fairness.
+    For example, there is the concept of individual fairness (where decisions are evaluated
+    on the level of individuals rather than groups), counter-factual fairness (e.g. does the
+    decision change if an individual's gender is changed from male to female), and many
+    others.
+    We would welcome contributions which enabled Fairlearn to help address some of these other harms.
+    Beyond these, there are also fairness concepts which are not amenable to
+    mathematical (and hence algorithmic) expression - justice, due process and
+    righting historic iniquities for example.
+
+Can the Fairlearn library be used to detect bias in datasets?
+    We do not have concrete plans for this at the present time.
+
+Can the Fairlearn library recommend ways to make my model fairer?
+    Fairness is a social concept, and no technological solution can make
+    things fair automatically.
+    In order to use the Fairlearn library successfully, you first need to work out
+    what fairness means for your particular problem.
+    We aim to provide a rich discussion of the topic of fairness in machine learning
+    on this website, together with links to the literature.
+    Once you have determined what fairness means for your AI system,
+    you can then work out how to use the various algorithms and fairness metrics
+    which Fairlearn supports to develop models which are
+    *fairer by the standard you have chosen*.
```

Comment:
Should we add that fairness may not always be quanitifiable.

Reply:
Like on line 70?

Reply:
Right. It's a different answer. Duplication isn't a big deal here since we're really trying to make the message stick :-)

Reply:
Duplicated. Well, rephrased slightly.